scripts/ci/podman-test.sh

```diff
@@ -1,30 +1,6 @@
 #!/bin/bash
 set -x -e -o pipefail
 
-if [ ! -e /etc/lsb-release ]; then
-       # This expects to run on Ubuntu
-       exit 1
-fi
-
-#shellcheck disable=SC1091
-. /etc/lsb-release
-
-echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${DISTRIB_RELEASE}/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
-
-curl -sL "https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_${DISTRIB_RELEASE}/Release.key" | apt-key add -
-
-# podman conflicts with a man page from docker-ce
-# this is a podman packaging bug (https://github.com/containers/libpod/issues/4747)
-apt-get -y purge docker-ce || :
-
-./apt-install \
-    apt-transport-https \
-    ca-certificates \
-    curl \
-    software-properties-common
-
-./apt-install podman containernetworking-plugins
```

Comment:
In the [installation script](https://github.com/actions/virtual-environments/blob/main/images/linux/scripts/installers/containers.sh) only `podman` is installed. Do we need `containernetworking-plugins` as well?

Reply:
The test seems to run happily without it. So I would say we do not need it.